scripts/build/debug/300-gdb.sh

```diff
@@ -122,6 +122,14 @@ do_debug_gdb_build()
 
         native_extra_config+=("--program-prefix=")
 
+        # Starting from GDB 11.x, gmp is needed as a dependency to build full
+        # gdb. And if target GMP gets built, explicitly point to installed library,
+        # as otherwise host library might be attempted to be used for target binary
+        # linkage.
+        if [ "${CT_GMP_TARGET}" = "y" ]; then
+            native_extra_config+=("--with-libgmp-prefix=${CT_SYSROOT_DIR}")
+        fi
+
```

Comment:
I wonder if this should be handled like zstd below i.e. `--with-libgmp=${complibs}`. It's slightly different because zstd is optional but GMP is required.

Reply:
IIRC it's `--with-libgmp-prefix`

Reply:
> I wonder if this should be handled like zstd below i.e. `--with-libgmp=${complibs}`. It's slightly different because zstd is optional but GMP is required.

Well, that's how it's done in Buildroot, see https://git.buildroot.net/buildroot/tree/package/gdb/gdb.mk#n161.
Do you want me to try something else?